tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/builtinoperations/CloneOperationTest.java

```diff
@@ -317,11 +317,11 @@ public void testCloneMap() {
 
     @Test
     public void testCloneNegative() {
-        Assert.assertEquals(negativeResult.getErrorCount(), 3);
+        Assert.assertEquals(negativeResult.getErrorCount(), 1);
         BAssertUtil.validateError(negativeResult, 0, "too many arguments in call to 'clone()'", 19, 13);
-        BAssertUtil.validateError(negativeResult, 1, "incompatible types: expected 'anydata', found 'typedesc<int>'",
-                                  24, 23);
-        BAssertUtil.validateError(negativeResult, 2, "incompatible types: expected 'anydata', found 'error'", 29, 15);
+//        BAssertUtil.validateError(negativeResult, 1, "incompatible types: expected 'anydata', found 'typedesc<int>'",
+//                                  24, 23);
+//        BAssertUtil.validateError(negativeResult, 2, "incompatible types: expected 'anydata', found 'error'", 29, 15);
```

Comment:
Why are we commenting these out? These are valid errors, right?

Reply:
Since `Cloneable` support `readonly` we support type we support `typedesc` and `error` as selectively immutable basic types. These are not errors now right ?

Reply:
Ah yes, my bad. Forgot we were changing the param type for these methods.

Reply:
We should remove these though. Instead of commenting them out.